src/main/kotlin/net/horizonsend/ion/Ion.kt

```diff
@@ -84,5 +93,12 @@ class Ion: JavaPlugin() {
 			wooltype.addIngredient(1, it)
 			server.addRecipe(wooltype)
 		}
+
+		val gunpowderrecipe = ShapelessRecipe(NamespacedKey(this, "gunpowderrecipe"), ItemStack(Material.GUNPOWDER))
```

Comment:
The namespaced key "gunpowderrecipe" can't be used multiple time otherwise it will create conflicts, give it a name like "gunpowdershapelessrecipe".

Reply:
Actually just remove the shaped recipe.